pytorch_lightning/utilities/imports.py

```diff
@@ -85,6 +85,42 @@ def _compare_version(package: str, op: Callable, version: str, use_base_version:
     return op(pkg_version, Version(version))
 
 
+class _RequirementAvailable:
+    """Boolean-like class for check of requirement with extras and version specifiers.
+
+    >>> _RequirementAvailable("torch>=0.1")
+    Requirement 'torch>=0.1' met
+    >>> bool(_RequirementAvailable("torch>=0.1"))
+    True
+    >>> bool(_RequirementAvailable("torch>100.0"))
+    False
+    """
+
+    def __init__(self, requirement: str) -> None:
+        self.requirement = requirement
+
+    def _check_requirement(self) -> None:
+        if not hasattr(self, "available"):
+            try:
+                pkg_resources.require([self.requirement])
```

Comment:
Docs for this say:

> require(*requirements)
> [...]
> requirements must be a string or a (possibly-nested) sequence thereof, specifying the distributions and versions required.

https://setuptools.pypa.io/en/latest/pkg_resources.html

So shouldn't it be:

```suggestion
                pkg_resources.require(self.requirement)
```